logic/transaction.js

```diff
@@ -1138,11 +1128,11 @@ class Transaction {
 		}
 
 		if (transaction.amount) {
-			transaction.amount = parseInt(transaction.amount);
+			transaction.amount = new bignum(transaction.amount);
```

Comment:
`transaction.amount = new bignum(transaction.amount || 0);` and remove conditions